ui/app/src/components/currencyPairPanel/Index.vue

```diff
@@ -3,7 +3,7 @@
 <template>
   <div class="field-wrappers">
     <CurrencyField
-      label="Input"
+      label="To"
```

Comment:
Hmm I think we're gonna need to pass these labels in to the `CurrencyPairPanel` and give them defaults of Input.

Reply:
Will fix this now